transformer_engine/jax/flax/module.py

```diff
@@ -739,6 +739,9 @@ class LayerNormMLP(TransformerEngineBase):
     activations: Sequence[Union[str, Callable]], default = ('relu',)
         The sequence of activation functions to apply after the first linear transformation.
         Each activation has its own transformation layer.
+    intermediate_dropout_rng_name: str, default = 'dropout'
+        The key in given RNGs via flax.linen.Module.apply that for
+        generating Dropout masks in the Multi-Head Attention.
```

Comment:
Is `intermediate_dropout_rng_name` for `Multi-Head Attention`?

Reply:
It is typo, fixed.